src/components/Nav/NewCollective.vue

```diff
@@ -33,51 +33,60 @@
 			</NcActionButton>
 		</template>
 	</NcAppNavigationItem>
-	<div v-else class="collective-create">
-		<form v-show="editing" @submit.prevent.stop="createCollective">
```

Comment:
Removing the `<form>` element with its submit event means that the form cannot be submitted by pressing <kbd>Return</kbd> anymore.

Reply:
Reverted, does work now :)